### PR TITLE
issue #12 : Adds labels defined in config file to state panel

### PR DESCRIPTION
**Pull request by Pedro12909**
I fixed the State Panel's formatting. All labels defined in the config.properties file are now properly shown on screen. Keep in mind that this still doesn't support messages and signals. I can work on that, but first I still need to research a little bit on how that works.

Here's a preview:
image

### CODE REVIEW

**Comment by Pedro12909** [comment]
Woops. Apparently I referenced the wrong issue in the commit message. Let me know if you want me to correct it.

This is the proper issue
https://image.ibb.co/gThXxn/Screenshot_from_2018_03_08_17_48_13.png

File: src/main/java/com/ksatstuttgart/usoc/gui/setup/GuiBuilder.java [hunk 1]
```diff
@@ -35,27 +35,14 @@
...
-import javafx.scene.control.Menu;
-import javafx.scene.control.MenuBar;
-import javafx.scene.control.MenuItem;
-import javafx.scene.control.RadioMenuItem;
-import javafx.scene.control.ScrollPane;
-import javafx.scene.control.SeparatorMenuItem;
-import javafx.scene.control.SplitPane;
-import javafx.scene.control.Tab;
-import javafx.scene.control.TabPane;
+import javafx.scene.control.*;
```
Comment (vstarlinger) [inline src/main/java/com/ksatstuttgart/usoc/gui/setup/GuiBuilder.java]: Heres the wild card import again 😅

File: src/main/java/com/ksatstuttgart/usoc/gui/setup/GuiBuilder.java [hunk 2]
```diff
@@ -184,64 +171,60 @@ public void handle(ActionEvent actionEvent) {
         return menuBar;
     }
 
-
-    /**
-     * Responsible for creating the State Panel
```
Comment (vstarlinger) [inline src/main/java/com/ksatstuttgart/usoc/gui/setup/GuiBuilder.java]: Why did you remove this comment?
